packages/p2p/src/components/app.jsx

```diff
@@ -167,7 +167,8 @@ class App extends React.Component {
                 if (old_order.status !== new_order.status) {
                     if (notification) {
                         // If order status changed, notify the user.
-                        notification.is_seen = is_current_order;
+                        notification.is_seen =
+                            is_current_order || new_order.status === 'timed-out' || new_order.status === 'blocked';
```

Comment:
There is no need to explicitly define the type of status, it always notifies the user on any status change.